ocaml/src/albamgr_protocol.ml

```diff
@@ -1017,12 +1040,12 @@ module Protocol = struct
            Nsm_host.to_buffer
            Llio.int64_to)
     | ListAvailableOsds ->
-      Llio.counted_list_to Osd.to_buffer
+      Llio.counted_list_to (OsdInfo.to_buffer ~version:2)
```

Comment:
There are no old clients for this call in the compatibility tests. only old servers.